src/pages/Dashboard/Apps/Add/index.jsx

```diff
@@ -263,35 +215,57 @@ export default class AppAdd extends Component {
             >
               {t('Deploy Test')}
             </Button>
-            <Button
-              onClick={() => {
-                if (isCreateApp) {
-                  history.replace(
-                    `/dashboard/app/${appDetail.app_id}/create-version`
-                  );
-                } else {
-                  appCreateStore.reload({ isCreateApp, appId });
-                }
-              }}
-              className={styles.addBtn}
-            >
-              {t('Add delivery type')}
-            </Button>
+            {!isAddVersion && (
+              <Button
+                onClick={() => {
```

Comment:
onClick handler should use instance method, so the button event handler function will not be created again and again.